Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "trauma"
-version = "2.1.1"
+version = "2.2.0"
```

Comment:
Don't update the version number. We change it when we cut a new release.

Reply:
I am using rust stable.
The format is mainly related to rustfmt.
Can you provide the relevant configuration?

```json
    "rust-analyzer.rustfmt.extraArgs": [
        "--config",
        "max_width=85,single_line_if_else_max_width=80,chain_width=32,format_code_in_doc_comments=false,imports_granularity=Crate"
    ],
    "rust-analyzer.checkOnSave.command": "clippy",
```